server/applicationInfo.ts

```diff
@@ -2,11 +2,30 @@ import fs from 'fs'
 import path from 'path'
 import config from './config'
 
-const packageJson =
```

Comment:
there was some hackiness here as during the relative location of package.json would vary dependent on whether we were running tests (compiled on fly) vs running the app (compiled to dist folder)

This is no longer needed due to use of DI